BCP195bis/draft-ietf-uta-rfc7525bis.md

```diff
@@ -349,17 +341,17 @@ Session resumption drastically reduces the number of TLS handshakes and thus is
 performance feature for most deployments.
 
 Stateless session resumption with session tickets is a popular strategy. For TLS 1.2, it is specified in
-{{?RFC5077}}.  For TLS 1.3, an equivalent PSK-based mechanism is described in
-Section 4.6.1 of {{RFC8446}}.
+{{?RFC5077}}.  For TLS 1.3, a more secure PSK-based mechanism is described in
+Section 4.6.1 of {{RFC8446}}. See [this post](https://blog.filippo.io/we-need-to-talk-about-session-tickets/) for a comparison of TLS 1.2 and 1.3 session resumption, and {{Springall16}} for a quantitative study of TLS cryptographic "shortcuts", including session resumption.
```

Comment:
Nice, I didn't know you could do that with kdrfc.  And since we are exchanging tips:
```suggestion
{{Section 4.6.1 of RFC8446}}. See [this post](https://blog.filippo.io/we-need-to-talk-about-session-tickets/) for a comparison of TLS 1.2 and 1.3 session resumption, and {{Springall16}} for a quantitative study of TLS cryptographic "shortcuts", including session resumption.
```